Assets/MRTK/Tools/AssetSwapUtility/AssetSwapUtility.cs

```diff
@@ -0,0 +1,388 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using UnityEngine;
+
+#if UNITY_EDITOR
+using UnityEditor;
+#endif // UNITY_EDITOR
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    /// <summary>
+    /// A scriptable object which contains data and UI functionality to swap object (asset) references within a scene.
+    /// </summary>
+    [CreateAssetMenu(fileName = "AssetSwapCollection", menuName = "Mixed Reality Toolkit/AssetSwapCollection")]
+    public class AssetSwapUtility : ScriptableObject
```

Comment:
It could be useful to have a menu item in the Mixed Reality Toolkit menu as easy entry point:

![image](https://user-images.githubusercontent.com/53493796/106317294-f6906380-6222-11eb-8419-e3c9165a86bd.png)

Reply:
Great idea! Just added one: 
![image](https://user-images.githubusercontent.com/13305729/106320376-a7006680-6227-11eb-885d-ea781ac3afad.png)